### DISCUSSION TITLE
`sort` field is visible after creating a M2M field

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17172] [Feature Requests]
When creating a M2M field the `sort` field is visible on the detail, but it should be hidden by default.

### To Reproduce

Create a new collection `projects` and within that collection a M2M field `tags`. Enable a `sort` field for the junction collection `projects_tags` while creating the M2M field. Navigate to the junction collection and you will find the visible `sort` field.

**Comment by ghost** [comment]
@rijkvanzanten You’ve converted this (https://github.com/directus/directus/issues/17153#event-8229960023) from an issue to a discussion and I was wondering if I expressed myself wrong as I am convinced it is a bug.

Here is a video of the described behavior.

https://user-images.githubusercontent.com/2291224/212561809-4e0f6b53-a0ef-4616-a745-27aed2743b37.mp4

**Comment by rijkvanzanten** [comment]
It wasn't originally built to be hidden by default, and the sort field is "just" a numeric field, hence I'm treating it as a request for improvement [^1]. I do agree it would be nice to just hide it by default (like we are with the foreign keys) as it's not really something we expect people to manually adjust (and you can always make it visible if that's what you wanna do anyways). 🙂 

[^1]: To keep a little order in the chaos that's OSS, we're always starting improvements/feature requests as a Discussion; once the Discussion reaches a point where we can conclusively say what / how we want to implement it, we make an Issue. This way, Issues are actionable items, and Discussions are.. discussions :) For these smaller tweaks, it can feel a little cumbersome, but it really does allow the maintainers to not lose their minds 😅

**Comment by rijkvanzanten** [comment]
This feature request has been accepted and is queued to be implemented! You can follow along with the progress here: https://github.com/directus/directus/issues/17238
